scripts/cloudformation_coverage.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+import importlib
+import json
+import mock
+import requests
+
+import moto
+
+# Populate CloudFormationModel.__subclasses__()
+moto.mock_all()
+
+
+def check(condition):
+    if bool(condition):
+        return "x"
+    else:
+        return " "
+
+
+def is_implemented(model, method_name):
+    # method_naem in model.__dict__ will be True if the method
+    # exists on the model and False if it's only inherited from
+    # CloudFormationModel.
+    return hasattr(model, method_name) and method_name in model.__dict__
+
+
+class CloudFormationChecklist:
+    def __init__(self, resource_name, schema):
+        self.resource_name = resource_name
+        self.schema = schema
+
+    def __str__(self):
+        missing_attrs_checklist = "\n".join(
+            [f"      - [ ] {attr}" for attr in self.missing_attrs]
+        )
+        report = (
+            f"- {self.resource_name}:\n"
+            f"   - [{check(self.creatable)}] create implemented\n"
+            f"   - [{check(self.updatable)}] update implemented\n"
+            f"   - [{check(self.deletable)}] delete implemented\n"
+            f"   - [{check(not self.missing_attrs)}] Fn::GetAtt implemented\n"
+        ) + missing_attrs_checklist
+
+        return report.strip()
+
+    @property
+    def service_name(self):
+        return self.resource_name.split("::")[1].lower()
+
+    @property
+    def model_name(self):
+        return self.resource_name.split("::")[2]
+
+    @property
+    def moto_model(self):
+        for subclass in moto.core.common_models.CloudFormationModel.__subclasses__():
+            subclass_service = subclass.__module__.split(".")[1]
+            subclass_model = subclass.__name__
+
+            if subclass_service == self.service_name and subclass_model in (
+                self.model_name,
+                "Fake" + self.model_name,
+            ):
+                return subclass
+
+    @property
+    def expected_attrs(self):
+        return list(self.schema.get("Attributes", {}).keys())
+
+    @property
+    def missing_attrs(self):
+        missing_attrs = []
+        for attr in self.expected_attrs:
+            try:
+                # TODO: Change the actual abstract method to return False
```

Comment:
I'm only patching this here because I think changing the actual abstract method to return False could maybe be considered a breaking change. But I do think it's more accurate to default to returning False than it is to default to returning True.